src/commands/show.ts

```diff
@@ -16,16 +17,17 @@ export async function showProblem(node?: LeetCodeNode): Promise<void> {
     if (!node) {
         return;
     }
-    await showProblemInternal(node.id);
+    await showProblemInternal(node);
```

Comment:
Though type casting from `LeetCodeNode` to `IProblem` is fine. Let's add getter in `LeetCodeNode` to get the data, which is typed as `IProblem`.

> It looks like I need to do some refactors here later after this PR, The definition of `LeetCodeNode` and `IProblem` is overlapped.